carbon-design-system/carbon-addons-iot-react
# feat(icon-switch): design updates

Closes #1536 

**Summary**

- Bring IconSwitch into alignment with new styles. 

[Icon content switcher - Style.pdf](https://github.com/carbon-design-system/carbon-addons-iot-react/files/7139752/Icon.content.switcher.-.Style.pdf)


**Change List (commits, features, bugs, etc)**

- update styles
- add disabled/children props to IconSwitch
- update stories to include more icons and show the divider
- add new stories

**Acceptance Test (how to verify the PR)**

- confirm styles match those shown in the PDF above

**Regression Test (how to make sure this PR doesn't break old functionality)**

- confirm this doesn't break interactions or designs in other stories that use the switchers. 
- DashboardEditor
- ImageGalleryModal
- HotspotTextStyleTab



**Things to look for during review**
